drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java

```diff
@@ -28,7 +29,7 @@ public class ChromeDriverBinaryHandler extends AbstractBinaryHandler {
     private static final String CHROME_DRIVER_VERSION_PROPERTY = "chromeDriverVersion";
     private static final String CHROME_DRIVER_URL_PROPERTY = "chromeDriverUrl";
 
-    public static final String CHROME_DRIVER_BINARY_NAME = "chromedriver" + (PlatformUtils.isWindows() ? ".exe" : "");
+    public static final Supplier<String> CHROME_DRIVER_BINARY_NAME = () -> "chromedriver" + (PlatformUtils.isWindows() ? ".exe" : "");
```

Comment:
Needs to be polished. Was mainly used for fixing tests, but should not touch the application code because of tests. We need to find a place in tests, where the OS is changed to Win and not properly returned to the initial state.